.github/pull_request_template.md

```diff
@@ -0,0 +1,13 @@
+### Pull Request Checklist
+
+#### [] Description
```

Comment:
Can we remove the checkboxes here? I don't think it makes sense in a heading and I'm not sure GitHub would render it.